### PR TITLE
Backend version

### LINKED ISSUE #477: ADD Backends version reporting

### PR DESCRIPTION
**Pull request by DvirDukhan**
this PR modifies `AI.INFO` arguments to optional. a call to `AI.INFO` without arguments will yields:
1. module version
2.  LLAPI version
3. RDB encoder version.
4. optional: TF version (if loaded)
5. optional onnx version (if loaded)

missing:
TFLIte version - not exported in current depdendecy structure
TORCH version - waiting for https://github.com/pytorch/pytorch/issues/44365

closes #477

### CODE REVIEW

**Review by lantiga — APPROVED** [review]
LGTM

File: docs/commands.md [hunk 1]
```diff
@@ -677,7 +677,15 @@ _Arguments_
...
 _Return_
 
-An array with alternating entries that represent the following key-value pairs:
+For a module genernal information: An array with alternating entries that represent the following key-value pairs:
+
+* **Version**: a string showing the current module version.
+* **Low level API Version**:  a string showing the current module's low level api version.
+* **RDB Encoding version**: a string showing the current module's RDB encoding version.
+* **TensorFlow version**: a string showing the current loaded TesnorFlow backend version.
+* **ONNX version**: a string showing the current loaded ONNX Runtime backend version.
```
Comment (alonre24) [inline docs/commands.md:686]: What about pytorch?

File: src/redisai.c [hunk 2]
```diff
@@ -805,12 +805,71 @@ int RedisAI_ScriptScan_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **arg
     return REDISMODULE_OK;
 }
 
+void _RedisAI_Info(RedisModuleCtx *ctx) {
+    RedisModuleString *rai_version = RedisModule_CreateStringPrintf(
+        ctx, "%d.%d.%d", REDISAI_VERSION_MAJOR, REDISAI_VERSION_MINOR, REDISAI_VERSION_PATCH);
+    RedisModuleString *llapi_version =
```
Comment (alonre24) [inline src/redisai.c:811]: Should we update our LLAPI version after supporting async and DAG? currently it is defined as "1".

File: docs/commands.md (hunk 1 above)
Comment (DvirDukhan) [reply docs/commands.md:686]: see in the description, we are waiting for a PR to expose this on TORCH sie

**Review by alonre24 — APPROVED** [review]

**Pull request merged**